packages/nimble-components/src/table-column/date-text/tests/table-column-date-text.stories.ts

```diff
@@ -113,9 +167,181 @@ export const dateTextColumn: StoryObj<TextColumnTableArgs> = {
             description:
                 'Set this attribute to identify which field in the data record should be displayed in each column. The field values must be of type `number` and represent the number of milliseconds since January 1, 1970 UTC. This is the representation used by the `Date` type.',
             control: { type: 'none' }
+        },
+        format: {
```

Comment:
Could you add docs for the column's validity object field? It might be helpful to mention that a good way to debug invalid configuration is to use [MDN's Try It widget](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#try_it) or a browser console. The browser actually gives very helpful exception messages.